### PR TITLE
Template update

### PR DESCRIPTION
**Pull request by dynamite-bud**
This PR introduces:

1. Update to newer `wasmer.toml` file
2. Update owner to `wasmer`
3. Add `SCRIPT_NAME` env var in `command.annotations.wasi`

Also closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,34 +24,38 @@ $ rustup target add wasm32-wasi
...
+
+You can also use `wasmer run` to test things locally.
 
 ```console
-$ wasmer run-unstable .
+$ wasmer run . --net
 INFO run: wasmer_wasix::runners::wcgi::runner: Starting the server address=127.0.0.1:8000 command_name="server"
 ```
 
+> The `--net` flag is required to enable networking support in Wasmer.
```
Comment (Michael-F-Bryan) [inline README.md]:
```suggestion
> **Note:** The `--net` flag is required to enable networking support in Wasmer.
```

File: README.md [hunk 2]
```diff
@@ -24,34 +24,38 @@ $ rustup target add wasm32-wasi
...
 
 ```console
-$ wasmer login $MY_API_TOKEN
-$ wasmer publish .
+$ wasmer login
+$ wasmer publish
 ```
 
-You can also use `wasmer run-unstable` to test things locally.
+> Please update the default namespace in the `wasmer.toml` file before
```
Comment (Michael-F-Bryan) [inline README.md]: I'm guessing you meant *"before publishing"*?

```suggestion
> Please update the default namespace in the `wasmer.toml` file before publishing.
```

File: wasmer.toml [hunk 3]
```diff
@@ -14,5 +14,10 @@ abi = "wasi"
...
 runner = "https://webc.org/runner/wcgi"
-module = "wcgi"
-annotations = { wcgi = { dialect = "rfc-3875" } }
+module = "server"
+
+[command.annotations]
+wcgi = { dialect = "rfc-3875" }
+
+[command.annotations.wasi]
+env = ['SCRIPT_NAME=rust_wcgi']
```
Comment (Michael-F-Bryan) [inline wasmer.toml]: Would it make more sense for users if we had tables for both sets of annotations?

```suggestion
[command.annotations.wcgi]
dialect = "rfc-3875"

[command.annotations.wasi]
env = ['SCRIPT_NAME=rust_wcgi']
```

Or maybe we should inline them both?

```suggestion
[command.annotations]
wcgi = { dialect = "rfc-3875" }
wasi = { env = ['SCRIPT_NAME=rust_wcgi'] }
```

@dynamite-bud  I'll leave it up to you to decide which looks cleaner.

Comment (dynamite-bud) [reply wasmer.toml]: Well actually `wasmer app create` automatically formatted it

File: README.md (hunk 1 above)
Comment (dynamite-bud) [reply README.md]: Updated this

File: README.md (hunk 2 above)
Comment (dynamite-bud) [reply README.md]: Updated this

**Pull request merged**